### DISCUSSION TITLE
Why are exponents higher for dark mode?

### DISCUSSION THREAD

**Discussion by xi** [Myndex/SAPC-APCA#86] [Q&A] [answered]
When the background is brighter than the text, APCA uses the exponents 0.56 and 0.57. In reverse polarity/dark mode, it uses the exponents 0.65 and 0.62.

As far as I understand, higher exponents are usually used to model brighter viewing conditions. For example [CIECAM02](https://en.wikipedia.org/wiki/CIECAM02) has a value `c` ("impact of surrounding") that goes from 0.525 for dark conditions to 0.69 for average conditions. (`c` is only one of several factors to the exponent though).

I would have expected that dark mode corresponds to darker viewing conditions. First, because the background color itself is often a significant part of the lighting conditions, and second, because some software automatically switches to dark mode at night.

So why does APCA go in the opposite direction?

**Comment by Myndex** [comment]
> _So why does APCA go in the opposite direction?_

It actually doesn't, though it may seem that way.

This is a good questions Tobias, and a partial answer is there is more going on here, and it is not as simple as the "brighter - more". The assumed _condition_ is the average (bright) condition FOR ALL MODES.

The deeper answer is that the full SAPC has as many as 5 color inputs, some of which are for the surrounding conditions. For the basic model, the "common worst case" is assumed, and that is what is important for useable guidelines.

In other words, for the basic model, the average condition is built in, and we are not simply modeling perceptual lightness but the _contrast_, and this is different and results in different curve considerations.

This is related to your other recent question, I will answer more there.

**Comment by Myndex** [comment, marked as answer]
Hi @xi 

I realize that I may not always understand what you are actually asking. This may be the result of translation errors. When I realize the actual gist of your queries, I do try to come back and address them. Your question above is another apples/apricots example.

In your question, you are talking about ambient illumination, and more specifically the _general_ adaptation level. As I noted above, this is _built in_ to APCA, and applies to both light and dark modes.

The difference that you are _actually_ asking about though is NOT related to general adaptation, it is related more to local adaptation and contrast adaptation/contrast constancy, and _THOSE_ are substantially driven by spatial frequency.

Stevens indicates this spatial frequency relationship, citing a 0.33 power curve for large patches, and 0.5 for point-light (i.e. a bright star at night). Consider that in dark mode, text is more like point-light, and light mode is the opposite.

I hope this clarifies the matter—keep in mind the contrast curves and exponents are derived from real-world empirical measurements.
